etc/kayobe/kolla/config/prometheus/prometheus.yml.d/80-radosgw-exporter.yml

```diff
@@ -0,0 +1,21 @@
+# yamllint disable-file
+---
```

Comment:
Does this work if ``stackhpc_enable_radosgw_usage_exporter = false``?
We normally see issues with empty yaml files, wondering if we need this:
https://github.com/stackhpc/stackhpc-kayobe-config/blob/afd30ccff36e55eb97fa536e6b319503fbc33f7e/etc/kayobe/ssh.yml#L19

Reply:
@MoteHue From our CI tests, no error was occurred from ``stackhpc_enable_radosgw_usage_exporter = false``
I'd say it's safe.

Reply:
It works because it's not Ansible, it's just a prometheus config file using YAML